plugins/dialog/plugin.js

```diff
@@ -2210,7 +2210,8 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 			// So we need to invent a really funny way to make it work.
 			var myScrollHandler = function() {
 					scrollFunc();
-					arguments.callee.prevScrollHandler.apply( this, arguments );
+					// Replaced arguments.callee (#2741)
+					myScrollHandler.prevScrollHandler.apply( this, arguments );
```

Comment:
This code is executed only on IE < 7 (not supported) or in quirks mode (`env.ie6Compat = version < 7 || env.quirks;`) 😬 Please check manually (because we don't run unit tests in quirks mode) if it doesn't throw any errors in quirks mode. It should be executed when dialog is opened and window scrolled.

Reply:
I have already checked it and it works fine, but you may take another look.